app/page/info/tokens.tsx

```diff
@@ -146,59 +153,200 @@ function TokenInfoModal({
           <span>{formatUnitsToFixed(token.balance, token.decimals)}</span>{" "}
```

Comment:
nit: Refactor this after the first PR is merged.

Reply:
OK!